modules/installation-osp-describing-cloud-parameters.adoc

```diff
@@ -40,8 +41,27 @@ clouds:
       project_name: 'devonly'
       auth_url: 'https://10.10.14.22:5001/v2.0'
 ----
+// Potential spot to import OSP content
+. _Optional_: If your {rh-openstack} installation uses self-signed certificate authority (CA) certificates for endpoint authentication:
+.. Copy the certificate authority file to your machine.
+.. In the command line, run the following commands to add the machine to the certificate authority trust bundle:
++
+----
+$ sudo cp ca.crt.pem /etc/pki/ca-trust/source/anchors/
+$ sudo update-ca-trust extract
+----
+
+.. Add the `cacerts` key to `clouds.yaml`. The value must be a non-root accessible path to the CA certificate:
```

Comment:
I am confused by the path being relative. What does it have to be relative to?

```
a non-root accessible path to the CA certificate, absolute or relative to [...]
```

@iamemilio relative to what? To the clouds.yaml file? Or to CWD while running the installer?

Reply:
My assumption was the working directory, but good catch! It's not clear.

Reply:
It should be able to parse relative paths from the installer directory, but I think its safest to just use absolute paths

Reply:
I have run a couple tests. These are my findings: https://github.com/openshift/installer/pull/3356